README.md

```diff
@@ -1,5 +1,11 @@
 # Production Docker deployment for Mattermost
 
+## WARNING:
+
+The current state of this repository does not work out of the box as the server version 5.31+ require Postgres versions of 10 or higher.
```

Comment:
```suggestion
The current state of this repository doesn't work out-of-the box since Mattermost server v5.31+ requires PostgreSQL versions of 10 or higher.
```

Reply:
How do you feel about "PostgreSQL version 10" (-"of")?